### PR TITLE
Fix pipeline tests

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app/pages/view-solution/fixtures/publicSolution.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
       "answers": {
         "summary": "Write on Time is a Citizen-facing Appointments Management system specifically designed to reduce the number of DNAs in your practice.",
         "description": "a description",
-        "link": "link.com"
+        "link": "//www.link.com"
```
Comment (alibombaye) [inline app/pages/view-solution/fixtures/publicSolution.json]: why we doing this here now?

File: app/pages/view-solution/ui.test.js [hunk 2]
```diff
@@ -119,16 +119,11 @@ test('should display the solution description', async (t) => {
...
 });
 
 test('should navigate to the link address when clicking on the link in the solution description', async (t) => {
   await pageSetup(t, 'all');
-
-  await nock('http://localhost:8080')
-    .get('/api/v1/Solutions/link.com/Public')
-    .reply(200, {});
-
   const solutionDescriptionLink = Selector('div[data-test-id="view-section-question-link"]');
```
Comment (alibombaye) [inline app/pages/view-solution/ui.test.js]: take it the nock wasn't required then...

File: app/pages/view-solution/fixtures/publicSolution.json (hunk 1 above)
Comment (kat1906) [reply app/pages/view-solution/fixtures/publicSolution.json]: to make it work? and work properly 😄

File: app/pages/view-solution/ui.test.js (hunk 2 above)
Comment (kat1906) [reply app/pages/view-solution/ui.test.js]: isn't needed if you redirect to www.link.com as it should

**Pull request merged**